### ISSUE TITLE
Extend tool to look up legacy AWS integration secrets

### ISSUE THREAD

**Issue by kelvintaywl** [CircleCI-Public/CircleCI-Env-Inspector#52]
#### context

Customers may have added AWS credentials through CircleCI's legacy AWS integration.
This can be confirmed when we inspect the `.aws` block when hitting the v1.1 API endpoint for project settings as follows:

```console
$ curl -H "Circle-Token: $CIRCLE_TOKEN" https://circleci.com/api/v1.1/project/<vcs>/<org>/<project>/settings | jq ".aws."

{
  "keypair" : {
    "access_key_id" : "xxxxABCD",
    "secret_access_key" : "xxxxdefg"
  }
}  
```

To delete these AWS credentials, customers can follow this Support Article:
https://support.circleci.com/hc/en-us/articles/11990015505947-How-to-remove-legacy-AWS-integration-secrets

Since the January 4th security incident, some customers may have also received emails from AWS (titled `[Action Required] CircleCI Security Alert to Rotate Access Keys`) [from January 7th onwards](https://circleci.com/blog/jan-4-2023-incident-report/#:~:text=On%20January%207%2C%202023%2C%20at%2018%3A30%20UTC%2C%20we%20began%20working%20with%20our%20partners%20at%20AWS%20to%20notify%20customers%20of%20potentially%20affected%20AWS%20tokens.%20We%20understand%20that%20those%20notifications%20were%20complete%20as%20of%20January%2012%2C%202023%2C%20at%2000%3A00%20UTC.).

These AWS access keys mentioned in the emails can refer to the credentials added via the legacy AWS integration in CircleCI as well.

#### ask

Would it be possible for us to update this tool to:

1. also look up for these AWS credentials set up in the legacy feature per project?

I understand we only look for project environment variables at the project level, currently:
https://github.com/CircleCI-Public/CircleCI-Env-Inspector/blob/027538c2ea649c6a57bbba99cb4e72b127fe367a/src/utils/circleci.ts#L133-L149

**Comment by kelvintaywl** [comment]
[side-notes]
sharing, as a 🇯🇵 customer also wrote a related article on how to find these AWS credentials set up in the legacy CircleCI AWS integration feature:
https://qiita.com/sot528/items/1c2cd72904e8527f5d38

**Comment by KyleTryon** [comment]
Solved in #56 for version 2 coming out today.
